source/extensions/filters/http/dynamo/dynamo_stats.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "envoy/stats/scope.h"
+
+#include "common/stats/symbol_table_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Dynamo {
+
+class DynamoStats {
+public:
+  DynamoStats(Stats::Scope& scope, const std::string& prefix);
+  Stats::SymbolTable& symbolTable() { return scope_.symbolTable(); }
+
+  Stats::Counter& counter(const std::vector<Stats::StatName>& names);
+  Stats::Histogram& histogram(const std::vector<Stats::StatName>& names);
+
+  /**
```

Comment:
Can we de-dup this comment and either have it here or in the other place with any relevant fixes?